components/player-view.js

```diff
@@ -135,7 +135,7 @@ module.exports = class Player extends React.Component {
     window.AppData.set({loaded: true, exited: false});
     if (this.props.queue[0].live) this.setState({time: 'LIVE'});
     if (duration) {
-      window.AppData.set({duration: duration});
+      window.AppData.set({duration});
```

Comment:
Yeah, https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand
And the ESLint rule is http://eslint.org/docs/rules/object-shorthand

Pretty sweet.